tests/005_define.test

```diff
@@ -18,7 +26,7 @@ meta-product = \${TEST_DEFINE}
 # Test substitution in a resource name
 file-resource "\${TEST_DEFINE}" {
         # Test substitution in the middle of a string
-        host-path = "$TESTS_DIR/\${TEST_DEFINE}"
+        host-path = "$TEST_PATH"
```

Comment:
We probably need to dig in more to see if we can make that possible. On Windows, an absolute file path has to start with a drive letter instead of a slash, so I don't see how those would be compatible on both systems, even if we supports both kinds of slashes.